src/psfmachine/machine.py

```diff
@@ -957,16 +959,10 @@ def build_shape_model(
             Keyword arguments to be passed to `_get_source_mask()`
         """
 
-        # gaia estimate flux values per pixel to be used as flux priors
-        self.source_flux_estimates = np.copy(np.asarray(self.sources.phot_g_mean_flux))
-
         # Mask of shape nsources x number of pixels, one where flux from a
         # source exists
-        if not hasattr(self, "source_mask"):
-            self._get_source_mask(**kwargs)
-        # Mask of shape npixels (maybe by nt) where not saturated, not faint,
-        # not contaminated etc
-        self._get_uncontaminated_pixel_mask()
+        # if not hasattr(self, "source_mask"):
+        self._get_source_mask(**kwargs)
```

Comment:
we need to compute the source masks again in order to account for a possible new `flux_cut_off` value that controls how deep the mask foes into the background level.